libnmstate/nm/applier.py

```diff
@@ -390,7 +391,11 @@ def _build_connection_profile(
         ovs_port_options = iface_desired_state.get(OvsB.OPTIONS_SUBTREE)
         settings.append(ovs.create_port_setting(ovs_port_options))
     elif iface_type == ovs.INTERNAL_INTERFACE_TYPE:
-        settings.append(ovs.create_interface_setting())
+        patch_state = iface_desired_state.get(OVSInterface.CONFIG_SUBTREE)
+        settings.append(ovs.create_interface_setting(patch_state))
```

Comment:
You may change this to
```
settings.extend(ovs.create_interface_setting(iface_desired_state))
```
And hide the detail of OVS patch information.

Reply:
Sure, will do!

Reply:
Done!